pkg/apis/apps/v1alpha1/feedinventory.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2022 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Important: Run "make generated" to regenerate code after modifying this file
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope="Namespaced",shortName=wg,categories=clusternet
+
+// FeedInventory defines a group of feeds which correspond to a subscription.
+type FeedInventory struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Feeds []FeedOrder `json:"feeds"`
+}
+
+// FeedOrder defines an abstract representation of a feed.
+type FeedOrder struct {
+	// Reference refers to the feed object of subscription.
+	//
+	// +required
+	// +kubebuilder:validation:Required
+	Reference Feed `json:"reference"`
```

Comment:
How about changing to inline struct, such as 
Feed `json:",inline"`

Reply:
Sure.